pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -633,6 +636,9 @@ func (w *watcher) commitFiles(ctx context.Context, latestData, eventName, commit
 
 	if commitMsg == "" {
 		commitMsg = fmt.Sprintf(defaultCommitMessageFormat, latestData, eventName)
+	} else {
+		rules := map[string]string{"Value": latestData, "EventName": eventName}
+		commitMsg = parseTemplate(commitMsg, rules)
 	}
```

Comment:
How about like this?

```suggestion
	commitMsg = parseCommitMsg(commitMsg, latestData, eventName)
```

Reply:
OK. Thank you for your review!